mlos_bench/mlos_bench/optimizers/convert_configspace.py

```diff
@@ -77,6 +99,7 @@ def _tunable_to_configspace(
     cs : ConfigurationSpace
         A ConfigurationSpace object that corresponds to the Tunable.
     """
+    # pylint: disable=too-complex
```

Comment:
still need this?

Reply:
yes, unfortunately. we have one `if` too many in the method